app/javascript/modules/listings/SharedHelpers.tsx

```diff
@@ -89,7 +89,9 @@ export const getEventNote = (listingEvent: ListingEvent) => {
     <div className="flex flex-col">
       {listingEvent.Venue && (
         <span className="links-space translate">
-          <Markdown>{listingEvent.Venue}</Markdown>
+          <Markdown options={{ namedCodesToUnicode: { "#39": "\u0027" } }}>
```

Comment:
> Curious at to why this does not use the renderMarkup function defined below?

I think it should, I've made an update to do that.

Reply:
> Should this options object, and the ones below, be extracted into some consts file so that we have better consistency?

I think we're okay for now, as long as we keep using the `renderMarkup` functions instead of `<Markdown>` directly. We may want to revisit this if we need to pass more options to `<Markdown>`.